src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -206,9 +206,9 @@ void AdaptDispatch::_WriteToBuffer(const std::wstring_view string)
 // - distance - Distance to move
 // Return Value:
 // - True.
```

Comment:
Now that everything is `void`, there's a whole bunch of these comments that should probably be nuked.